x/liquidity/module.go

```diff
@@ -126,9 +127,7 @@ func (AppModule) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 }
 
 // RegisterInvariants registers the liquidity module invariants.
-func (am AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
-	keeper.RegisterInvariants(ir, am.keeper)
```

Comment:
Could you give a few details why this isn't required anymore?

Reply:
After the forced withdrawal, all pools will be removed, so escrow will no longer occur, so it was removed because there is no need to check the invariant